mantle/platform/api/do/api.go

```diff
@@ -49,6 +49,10 @@ type Options struct {
 	Size string
 	// Numeric image ID, {alpha, beta, stable}, or user image name
 	Image string
+	// Identifies if we are using IPv6 or not.
```

Comment:
@dustymabe added a comment here were we first use see the IPv6 boolean. Also renamed the variables to make it a bit more clear what is happening.

Reply:
Bikeshedding: it'd be clearer/more specific to say e.g. "Whether to allocate an IPv6 address to the droplet".